setup/environment-linux.yml

```diff
@@ -6,44 +6,42 @@ channels:
 dependencies:
   - _libgcc_mutex=0.1=conda_forge
   - _openmp_mutex=4.5=0_gnu
-  - blas=1.0=openblas
-  - blosc=1.16.3=hd408876_0
-  - bzip2=1.0.8=h7b6447c_0
-  - ca-certificates=2020.1.1=0
-  - certifi=2019.11.28=py38_0
+  - blas=1.1
+  - blosc=1.21.0
+  - bzip2=1.0.8
+  - ca-certificates
+  - certifi
   - coverage=5.5
-  - cython=0.29.15=py38he6710b0_0
-  - hdf5=1.10.4=hb1b8bf9_0
-  - intel-openmp=2020.0=166
+  - hdf5=1.10.5
+  - intel-openmp
   - ld_impl_linux-64=2.33.1=h53a641e_8
-  - libffi=3.2.1=he1b5a44_1006
-  - libgcc-ng=9.2.0=h24d8f2e_2
-  - libgfortran-ng=7.3.0=hdf63c60_0
-  - libgomp=9.2.0=h24d8f2e_2
-  - libopenblas=0.3.7=h5ec1e0e_6
-  - libstdcxx-ng=9.2.0=hdf63c60_2
-  - lz4-c=1.8.1.2=h14c3975_0
-  - lzo=2.10=h49e0be7_2
+  - libffi=3.2.1
+  - libgcc-ng
+  - libgfortran-ng
+  - libgomp
+  - libopenblas=0.3.13
+  - libstdcxx-ng
+  - lz4-c=1.9.3
+  - lzo=2.10
```

Comment:
Wouldn't it be better to include the exact version number of these dependencies.